client/src/components/Applet/Applet.js

```diff
@@ -7,7 +7,8 @@ import Highlights from './../Highlights/Highlights.js';
 import Details from './../Details/Details.js';
```

Comment:
Maybe you could find a more descriptive PR title- not sure what some behavior is :)